server/views/pages/timetable.njk

```diff
@@ -9,22 +9,57 @@
 
 {% block content %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
+
+      <nav aria-labelledby="selected-date">
+        <h2 class="govuk-heading-m" id="selected-date">{{ selectedDate | formatDate('EEEE d MMMM yyyy') }}</h2>
 
-        <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
+        <ul class="govuk-list bapv-timetable-dates govuk-!-margin-bottom-1">
+          {% for day in weekOfDates %}
+            {% set selected = day == selectedDate %}
+            <li class="bapv-timetable-dates__date {{ 'bapv-timetable-dates__date--selected' if selected }}">
+              <span>{{ day | formatDate('E') }}</span>
+                {% if not selected %}<a href="/timetable?date={{ day | formatDate('yyyy-MM-dd') }}">{% endif %}
+                  {{ day | formatDate('d') }}<span class="govuk-visually-hidden"> {{ day | formatDate('MMMM yyyy') }}</span>
+                {% if not selected %}</a>{% endif %}
+            </li>
+          {% endfor %}
+        </ul>
+        <ul class="govuk-list bapv-timetable-dates">
+          <li class="moj-pagination__item  moj-pagination__item--prev">
+            <a class="moj-pagination__link" href="/timetable?date={{ previousWeek | formatDate('yyyy-MM-dd') }}" data-test="previous-week">Previous week</a>
+          </li>
+          <li class="moj-pagination__item  moj-pagination__item--next">
+            <a class="moj-pagination__link" href="/timetable?date={{ nextWeek | formatDate('yyyy-MM-dd') }}" data-test="next-week">Next week</a>
+          </li>
+        </ul>
+      </nav>
+      
+    </div>
+  </div>
 
-        <h2 class="govuk-heading-m" data-test="date-heading">Monday DD MMMM 2023</h2>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <hr>
+      <p>schedule....</p>
```

Comment:
Is this meant to be here?

Reply:
Well, sort of - I had these in as a placeholder for a full-width column where the table will go (that Ben's working on). Feature flagged so only on dev - but commented out to look tidier 🙂